packages/cozy-harvest-lib/src/components/KonnectorManager.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import PropTypes from 'react-proptypes'
+
+import { withMutations } from 'cozy-client'
+
+import AccountCreationSuccess from './AccountCreationSuccess'
+import AccountForm from './AccountForm'
+import { accountsMutations } from '../connections/accounts'
+import { prepareAccountData } from '../helpers/accounts'
+import {
+  buildKonnectorCron,
+  buildKonnectorTriggerAttributes
+} from '../helpers/triggers'
+import { triggersMutations } from '../connections/triggers'
+
+const CREATION_SUCCESS = 'CREATION_SUCCESS'
+const CREATING = 'CREATING'
+const IDLE = 'IDLE'
+
+class KonnectorManager extends Component {
+  state = {
+    status: IDLE
+  }
+
+  onSubmit = async data => {
+    const {
+      createAccount,
+      createTrigger,
+      konnector,
+      launchTrigger,
+      onLoginSuccess,
+      waitForLoginSuccess
+    } = this.props
+    console.debug({ data })
+    this.setState({ status: CREATING })
+
+    const account = await createAccount(prepareAccountData(konnector, data))
+
+    const trigger = await createTrigger(
+      buildKonnectorTriggerAttributes({
+        konnector,
+        account,
+        cron: buildKonnectorCron(konnector)
+      })
+    )
+
+    let job = await launchTrigger(trigger)
+    job = await waitForLoginSuccess(job)
+
+    console.debug({ job })
+
+    if (['queued', 'running'].includes(job.state)) {
```

Comment:
What happens here if we have another `job.state` like error? Since this checking is done only in the onSubmit, it won't be handled? Or maybe it is like this for now.